docker-images/kafka-base/Dockerfile

```diff
@@ -1,41 +1,38 @@
-FROM centos:7
-
-ARG JAVA_VERSION
+FROM strimzi/java-base:latest
```

Comment:
Open question: What are the pros and cons of basing the `kafka-base` on the `java-base`?